CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Unreleased
 - Prevent Drop-in from firing requestable events prematurely (#511)
 - Add `off` method for unsubscribing from events without tearing down
 - Clean up extraneous css rules
+- Add 3DS version 2 support
```

Comment:
In our docs we now officially call it `3DS 2`: https://developers.braintreepayments.com/guides/3d-secure/migration/android/v3

Not to bike shed this more tho.